app/controllers/calculators/take_whole_pot_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Calculators
+  class TakeWholePotController < CalculatorsController
+    def show
```

Comment:
That occurred to me earlier.

We could set cache headers here too, however each result is likely to be a snowflake given different pot sizes and incomes values being passed in the querystring.